src/Lykke.Sdk/LykkeApplicationBuilderExtensions.cs

```diff
@@ -21,6 +21,16 @@ public static class LykkeApplicationBuilderExtensions
         /// </summary>
         /// <param name="app"></param>
         public static void UseLykkeConfiguration(this IApplicationBuilder app)
+        {
+            app.UseLykkeConfiguration(null);
+        }
+
+        /// <summary>
+        /// Configure Lykke service.
+        /// </summary>
+        /// <param name="app">IApplicationBuilder implementation.</param>
+        /// <param name="createError">Error creation handler.</param>
+        public static void UseLykkeConfiguration(this IApplicationBuilder app, CreateErrorResponse createError)
```

Comment:
I suggest to add ```LykkeConfigurationOptions``` class with ```CreateErrorResponse``` property and add ```Action<LykkeConfugrationOptions> configure``` as parameter of ```UseLykkeConfiguration``` method. This will allow as to add new options to configure in future without breaking changes

Reply:
Like this https://github.com/LykkeCity/Lykke.Logs/pull/46/files#diff-1e69354934d9cdb311c1eed2b66d5e49R15